src/sorting/python/merge_sort/src/merge_sort.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on 17 01 24
+
+@author: tvsirius
+"""
+
+
+def merge_sort(list_to_sort: list[any], ascending: bool = True) -> list[any]:
```

Comment:
I wasn't familiar with this function annotation, but thanks now I know

Reply:
Check constraints.md file. It has some explanation